pwa/src/components/translations/translationForm.js

```diff
@@ -186,7 +186,7 @@ export default function TranslationForm({ id }) {
                             id={"translationTableInput"}
                             data={translation && translation.translationTable && translation.translationTable}
                             nameOverride={"Table"}
-                            required={"true"} />
+                            required={true} />
```

Comment:
Tip: wanneer je in React een boolean prop meegeeft, dan hoef je alleen bij `false` de waarde te setten.

Dus: `required` === `required={true}`